flink-cdc-common/src/main/java/org/apache/flink/cdc/common/utils/DateTimeUtils.java

```diff
@@ -109,7 +119,7 @@ private static long internalParseTimestampMillis(String dateStr, String format,
         } catch (ParseException e) {
             LOG.error(
                     String.format(
-                            "Exception when parsing datetime string '%s' in format '%s'",
+                            "Exception when parsing datetime string '%s' in format '%s', the default value Long.MIN_VALUE(-9223372036854775808) will be returned.",
```

Comment:
`SystemFunctionUtils` (and `DateTimeUtils`) are evaluated inside Janino engine. Will these error messages ever be printed out?